### DISCUSSION TITLE
How to change language

### DISCUSSION THREAD

**Discussion by klorinczi** [sienori/Tab-Session-Manager#370] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Please allow changing the language in the Settings.

**Describe the solution you'd like**
Please allow changing the language in the Settings.

**Comment by sienori** [comment, marked as answer]
To change the language, please set the value of `intl.locale.requested` in `about:config`.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Testing_out_your_extension
